src/redux/constants/ActionTypes.js

```diff
@@ -0,0 +1,4 @@
+export const actionTypes = {
```

Comment:
Is there any reason to wrap actions inside `actionTypes` const as the file name says it will only contain redux actions ?